### PR TITLE
Potted plants are no longer prone to wear the One Ring when trimmed and going invisible as a result

### PR DESCRIPTION
**Pull request by GoldenAlpharex**
## About The Pull Request
Potted plants shouldn't become invisible after being trimmed anymore.

## Why It's Good For The Game
Invisible plants are bad, need I say more?

## Changelog
:cl: GoldenAlpharex
fix: Potted plants are no longer self-concious about their appearance changing when trimmed and will now hopefully properly display themselves as a result.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -353,11 +353,7 @@
 /obj/item/kirbyplants/proc/generate_states()
 	random_plant_states = list()
 	for(var/i in 1 to 25)
-		var/number
-		if(i < 10)
-			number = "0[i]"
-		else
-			number = "[i]"
+		var/number = "[i]"
 		random_plant_states += "plant-[number]"
```
Comment (Mothblocks) [inline code/game/objects/structures/flora.dm:357]:
```suggestion
		random_plant_states += "plant-[i]"
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]
I don't understand how this fixes it, but whatever

**Comment by Mothblocks** [comment]
you need to undraft your stuff

**Comment by AMonkeyThatCodes** [comment]
> I don't understand how this fixes it, but whatever

noob the icon states dont have 0 in em

**Comment by Fikou** [comment]
there are a lot of instances of potted plants on maps having var edited icon states. are you sure this doesnt fuck that up?

**Comment by GoldenAlpharex** [comment]
> there are a lot of instances of potted plants on maps having var edited icon states. are you sure this doesnt fuck that up?

I'll look it up tonight, I guess. So long as this gets re-opened, at least. Damn GitHub notifications

**Pull request closed without merging**